hack/config.sh

```diff
@@ -1,5 +1,5 @@
 binaries="cmd/virt-controller cmd/virt-launcher cmd/virt-handler cmd/virt-api cmd/virtctl"
-docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer"
+docker_images="$binaries images/haproxy images/iscsi-demo-target-tgtd images/vm-killer images/libvirtd"
```

Comment:
Travis is not the problem. It is when building the images inside our functional tests and in the development environment. These few packages are not that much an issue. We need to install a few hundred packages for libvirt alone, that is the bottleneck.